src/common/KokkosKernels_Utils.hpp

```diff
@@ -154,6 +155,104 @@ void get_suggested_vector_team_size(
 
 }
 
+#else
+template <typename idx, typename ExecutionSpace>
+void get_suggested_vector_size(
+    int &suggested_vector_size_,
+    idx nr, idx nnz){
+
+    suggested_vector_size_ =  1;
+
+#if defined( KOKKOS_ENABLE_SERIAL )
+  if (Kokkos::Impl::is_same< Kokkos::Serial , ExecutionSpace >::value){
+    suggested_vector_size_ =  1;
+  }
+#endif
+
+#if defined( KOKKOS_ENABLE_THREADS )
+  if (Kokkos::Impl::is_same< Kokkos::Threads , ExecutionSpace >::value){
+    suggested_vector_size_ =  1;
+  }
+#endif
+
+#if defined( KOKKOS_ENABLE_OPENMP )
+  if (Kokkos::Impl::is_same< Kokkos::OpenMP, ExecutionSpace >::value){
+    suggested_vector_size_ =  1;
```

Comment:
@srajama1 That's how it was beforehand, I just separated the function to get vector and team size into 2 separate parts. The vector size is determined first, then used in a dummy `team_policy` to get an estimate for the max team size. That is then used to tune the team size that is actually used in the `parallel_for` later. The changes were needed because the `team_size_max` call used to be static member function, but with deprecated code disabled it is now a member function that needs to be called from an instantiated policy (hence the need for the dummy team_policy).